gatt/gatt_linux.py

```diff
@@ -596,21 +596,28 @@ def read_value(self, offset=0):
             error = _error_from_dbus_error(e)
             self.service.device.characteristic_read_value_failed(self, error=error)
 
-    def write_value(self, value, offset=0):
+    def write_value(self, value, offset=0, with_response=True):
         """
         Attempts to write a value to the characteristic.
 
         Success or failure will be notified by calls to `write_value_succeeded` or `write_value_failed` respectively.
 
         :param value: array of bytes to be written
         :param offset: offset from where to start writing the bytes (defaults to 0)
+        :param with_response: whether a response should be requested for the write (defaults to True)
         """
         bytes = [dbus.Byte(b) for b in value]
 
+        if with_response == True:
+            write_type = "request"
+        else:
+            write_type = "command"
+
```

Comment:
`write_type = 'request' if with_response else 'command'`

might be more appropriate here, or if more explicit code is preferred:

```
if with_response:
    write_type = 'request'
else:
    write_type = 'command'
```